JSDemos/Demos/RangeSlider/Overview/React/App.js

```diff
@@ -99,7 +99,7 @@ class App extends React.Component {
                 max={100}
                 start={this.state.startValue}
                 end={this.state.endValue}
-                callValueChange='onMovingComplete'
+                valueChangeMode='onHandleRelease'
```

Comment:
```suggestion
                valueChangeMode="onHandleRelease"
```
Just for consistency

Reply:
Also line 66: why does it work? I mean the quotes instead of brackets

Reply:
@EugeniyKiyashko
[fix single brackets](https://github.com/DevExpress/devextreme-demos/pull/2205/commits/e743efc7b90b8640740820813111ab6fe4206588)